src/applications/facility-locator/components/Alert.jsx

```diff
@@ -21,11 +19,15 @@ const Alert = ({ title, description, displayType }) => {
 
   return (
     <div
-      className={`usa-alert ${cssClass} vads-u-margin-top--0 vads-u-margin-bottom--2 ${
-        isMobile
-          ? 'vads-u-margin-right--3 vads-u-margin-left--2 vads-u-width--auto'
-          : ''
-      }`}
+      className={`usa-alert ${cssClass}
+      vads-u-margin-top--0 
```

Comment:
These newlines will actually show up in the final markup I believe on the dom (doesn't really matter)

Reply:
Yeah, that's unfortunate. Maybe there's a way to remove them with WebPack?